Neo4j fails with "Some jar procedure files are invalid, see log for details" on CI

### Describe the bug

https://buildkite.com/stellar/stellargraph-public/builds/2732

### To Reproduce

Steps to reproduce the behavior:

1. Run a build on CI

### Observed behavior

Neo4j crashes on start-up which causes the notebook to fail.
```
2020-03-27T05:14:10.622684576Z Active database: graph.db
2020-03-27T05:14:10.628825992Z Directories in use:
2020-03-27T05:14:10.628841058Z   home:         /var/lib/neo4j
2020-03-27T05:14:10.628845748Z   config:       /var/lib/neo4j/conf
2020-03-27T05:14:10.628850353Z   logs:         /logs
2020-03-27T05:14:10.628854422Z   plugins:      /var/lib/neo4j/plugins
2020-03-27T05:14:10.628858143Z   import:       /var/lib/neo4j/import
2020-03-27T05:14:10.628861718Z   data:         /var/lib/neo4j/data
2020-03-27T05:14:10.628865585Z   certificates: /var/lib/neo4j/certificates
2020-03-27T05:14:10.628869302Z   run:          /var/lib/neo4j/run
2020-03-27T05:14:10.630536734Z Starting Neo4j.
2020-03-27T05:14:12.458205736Z 2020-03-27 05:14:12.455+0000 INFO  ======== Neo4j 3.5.17 ========
2020-03-27T05:14:12.474372134Z 2020-03-27 05:14:12.474+0000 INFO  Starting...
2020-03-27T05:14:16.171254080Z 2020-03-27 05:14:16.170+0000 ERROR Failed to start Neo4j: Starting Neo4j failed: Component 'org.neo4j.server.database.LifecycleManagingDatabase@5328a9c1' was successfully initialized, but failed to start. Please see the attached cause exception "Some jar procedure files are invalid, see log for details.". Starting Neo4j failed: Component 'org.neo4j.server.database.LifecycleManagingDatabase@5328a9c1' was successfully initialized, but failed to start. Please see the attached cause exception "Some jar procedure files are invalid, see log for details.".
2020-03-27T05:14:16.171283863Z org.neo4j.server.ServerStartupException: Starting Neo4j failed: Component 'org.neo4j.server.database.LifecycleManagingDatabase@5328a9c1' was successfully initialized, but failed to start. Please see the attached cause exception "Some jar procedure files are invalid, see log for details.".
2020-03-27T05:14:16.171289649Z 	at org.neo4j.server.exception.ServerStartupErrors.translateToServerStartupError(ServerStartupErrors.java:45)
2020-03-27T05:14:16.171294046Z 	at org.neo4j.server.AbstractNeoServer.start(AbstractNeoServer.java:187)
2020-03-27T05:14:16.171298306Z 	at org.neo4j.server.ServerBootstrapper.start(ServerBootstrapper.java:124)
2020-03-27T05:14:16.171302273Z 	at org.neo4j.server.ServerBootstrapper.start(ServerBootstrapper.java:91)
2020-03-27T05:14:16.171306144Z 	at org.neo4j.server.CommunityEntryPoint.main(CommunityEntryPoint.java:32)
2020-03-27T05:14:16.171310162Z Caused by: org.neo4j.kernel.lifecycle.LifecycleException: Component 'org.neo4j.server.database.LifecycleManagingDatabase@5328a9c1' was successfully initialized, but failed to start. Please see the attached cause exception "Some jar procedure files are invalid, see log for details.".
2020-03-27T05:14:16.171314473Z 	at org.neo4j.kernel.lifecycle.LifeSupport$LifecycleInstance.start(LifeSupport.java:473)
2020-03-27T05:14:16.171318195Z 	at org.neo4j.kernel.lifecycle.LifeSupport.start(LifeSupport.java:111)
2020-03-27T05:14:16.171321872Z 	at org.neo4j.server.AbstractNeoServer.start(AbstractNeoServer.java:180)
2020-03-27T05:14:16.171325618Z 	... 3 more
2020-03-27T05:14:16.171329204Z Caused by: java.lang.RuntimeException: Error starting org.neo4j.graphdb.facade.GraphDatabaseFacadeFactory, /data/databases
2020-03-27T05:14:16.171345852Z 	at org.neo4j.graphdb.facade.GraphDatabaseFacadeFactory.initFacade(GraphDatabaseFacadeFactory.java:232)
2020-03-27T05:14:16.171351977Z 	at org.neo4j.graphdb.facade.GraphDatabaseFacadeFactory.newFacade(GraphDatabaseFacadeFactory.java:148)
2020-03-27T05:14:16.171360363Z 	at org.neo4j.server.database.CommunityGraphFactory.newGraphDatabase(CommunityGraphFactory.java:41)
2020-03-27T05:14:16.171364328Z 	at org.neo4j.server.database.LifecycleManagingDatabase.start(LifecycleManagingDatabase.java:90)
2020-03-27T05:14:16.171368323Z 	at org.neo4j.kernel.lifecycle.LifeSupport$LifecycleInstance.start(LifeSupport.java:452)
2020-03-27T05:14:16.171372330Z 	... 5 more
2020-03-27T05:14:16.171376055Z Caused by: org.neo4j.kernel.lifecycle.LifecycleException: Component 'org.neo4j.kernel.impl.proc.Procedures@41477a6d' was successfully initialized, but failed to start. Please see the attached cause exception "Some jar procedure files are invalid, see log for details.".
2020-03-27T05:14:16.171380674Z 	at org.neo4j.kernel.lifecycle.LifeSupport$LifecycleInstance.start(LifeSupport.java:473)
2020-03-27T05:14:16.171384445Z 	at org.neo4j.kernel.lifecycle.LifeSupport.start(LifeSupport.java:111)
2020-03-27T05:14:16.171388208Z 	at org.neo4j.graphdb.facade.GraphDatabaseFacadeFactory.initFacade(GraphDatabaseFacadeFactory.java:227)
2020-03-27T05:14:16.171392778Z 	... 9 more
2020-03-27T05:14:16.171396479Z Caused by: java.util.zip.ZipException: Some jar procedure files are invalid, see log for details.
2020-03-27T05:14:16.171400367Z 	at org.neo4j.kernel.impl.proc.ProcedureJarLoader.loadProceduresFromDir(ProcedureJarLoader.java:76)
2020-03-27T05:14:16.171404127Z 	at org.neo4j.kernel.impl.proc.Procedures.start(Procedures.java:323)
2020-03-27T05:14:16.171407955Z 	at org.neo4j.kernel.lifecycle.LifeSupport$LifecycleInstance.start(LifeSupport.java:452)
2020-03-27T05:14:16.171411773Z 	... 11 more
2020-03-27T05:14:16.180892087Z 2020-03-27 05:14:16.180+0000 INFO  Neo4j Server shutdown initiated by request
```
### Expected behavior

Neo4j should start-up correctly and not crash, like in the earliest build before that https://buildkite.com/stellar/stellargraph-public/builds/2729

### Environment

Operating system: CI

Python version: 3.6

Package versions: 

<details>

```
absl-py==0.9.0
ansiwrap==0.8.4
appdirs==1.4.3
astor==0.8.1
attrs==19.3.0
backcall==0.1.0
black==19.10b0
bleach==3.1.4
boto3==1.12.30
botocore==1.15.30
cachetools==4.0.0
certifi==2019.11.28
chardet==3.0.4
Click==7.0
colorama==0.4.3
coverage==4.5.4
cycler==0.10.0
decorator==4.4.2
defusedxml==0.6.0
docopt==0.6.2
docutils==0.15.2
entrypoints==0.3
gast==0.2.2
gensim==3.8.1
google-api-core==1.16.0
google-auth==1.12.0
google-auth-oauthlib==0.4.1
google-cloud-core==1.3.0
google-cloud-storage==1.26.0
google-pasta==0.2.0
google-resumable-media==0.5.0
googleapis-common-protos==1.51.0
grpcio==1.27.2
h5py==2.10.0
idna==2.9
importlib-metadata==1.5.2
ipykernel==5.2.0
ipython==7.13.0
ipython-genutils==0.2.0
ipywidgets==7.5.1
isodate==0.6.0
jedi==0.16.0
Jinja2==2.11.1
jmespath==0.9.5
joblib==0.14.1
jsonschema==3.2.0
jupyter==1.0.0
jupyter-client==6.1.2
jupyter-console==6.1.0
jupyter-core==4.6.3
Keras-Applications==1.0.8
Keras-Preprocessing==1.1.0
kiwisolver==1.1.0
llvmlite==0.31.0
Markdown==3.2.1
MarkupSafe==1.1.1
matplotlib==3.2.1
mistune==0.8.4
more-itertools==8.2.0
mplleaflet==0.0.5
nbclient==0.1.0
nbconvert==5.6.1
nbformat==5.0.4
neobolt==1.7.16
neotime==1.7.4
networkx==2.4
notebook==6.0.3
numba==0.48.0
numpy==1.18.2
oauthlib==3.1.0
opt-einsum==3.2.0
packaging==20.3
pandas==1.0.3
pandocfilters==1.4.2
papermill==2.0.0
parso==0.6.2
pathspec==0.7.0
pexpect==4.8.0
pickleshare==0.7.5
pluggy==0.13.1
prometheus-client==0.7.1
prompt-toolkit==2.0.10
protobuf==3.11.3
ptyprocess==0.6.0
py==1.8.1
py-cpuinfo==5.0.0
py2neo==4.3.0
pyasn1==0.4.8
pyasn1-modules==0.2.8
Pygments==2.3.1
pyparsing==2.4.6
pyrsistent==0.16.0
pytest==5.3.1
pytest-benchmark==3.2.3
pytest-cov==2.8.1
python-dateutil==2.8.1
pytz==2019.3
PyYAML==5.3.1
pyzmq==19.0.0
qtconsole==4.7.2
QtPy==1.9.0
rdflib==4.2.2
regex==2020.2.20
requests==2.23.0
requests-oauthlib==1.3.0
rsa==4.0
s3transfer==0.3.3
scikit-learn==0.22.2.post1
scipy==1.4.1
seaborn==0.10.0
Send2Trash==1.5.0
six==1.14.0
smart-open==1.10.0
stellargraph==0.12.0b0
tenacity==6.1.0
tensorboard==2.1.1
tensorflow==2.1.0
tensorflow-estimator==2.1.0
termcolor==1.1.0
terminado==0.8.3
testpath==0.4.4
textwrap3==0.9.2
toml==0.10.0
tornado==6.0.4
tqdm==4.43.0
traitlets==4.3.3
treon==0.1.3
typed-ast==1.4.1
urllib3==1.24.3
wcwidth==0.1.9
webencodings==0.5.1
Werkzeug==1.0.0
widgetsnbextension==3.5.1
wrapt==1.12.1
zipp==3.1.0

```

</details>

### Additional context

Add any other context about the problem here. For example:

Data used: *if this occurs with a specific dataset, which dataset and how to acquire it*

Screenshots: *if applicable, add some screenshots to explain the behaviour*

--- a/docker/stellargraph-neo4j/Dockerfile
+++ b/docker/stellargraph-neo4j/Dockerfile
@@ -1,10 +1,8 @@
-FROM neo4j:3.5
+ARG NEO4J_VERSION
+FROM neo4j:${NEO4J_VERSION}
 
-SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
+ARG NEO4J_VERSION
 
 USER neo4j
-
-ENV APOC_VERSION="3.5.0.9" APOC_SHA="b4b1a8f8940ad250c17b45296459afcfff2eb1beee18b8c8a394e2b5a434183b924b34cb5b04fea1a371fbfb87a286228bd4ae814829e0ef99f9e190a764fa1d"
-RUN echo "--- downloading apoc jar" && \
-    wget --no-verbose "https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases/download/${APOC_VERSION}/apoc-${APOC_VERSION}-all.jar" --directory-prefix plugins/ && \
-    echo "${APOC_SHA}  ${PWD}/plugins/apoc-${APOC_VERSION}-all.jar" | sha512sum -c
+COPY install-apoc.sh /
+RUN bash /install-apoc.sh

